solana/ts/scripts/config/mainnet/quoter.json

```diff
@@ -9,32 +9,39 @@
       "gasCost": 500000,
       "wormholeTransceiverIndex": 0,
       "isSupported": true
+    },
+    {
+      "programId": "NttBm3HouTCFnUBz32fEs5joQFRjFoJPA8AyhtgjFrw",
+      "tokenAddress": "3dQTr7ror2QPKQ3GbBCokJUmjErGg8kTJzdnYjNfvi3Z",
+      "gasCost": 500000,
```

Comment:
I'm not familiar with this value, but is that okay? In what unit is that configured?

Reply:
gas units